pkg/workflows/create.go

```diff
@@ -359,9 +371,49 @@ func (s *DeleteBootstrapClusterTask) Run(ctx context.Context, commandContext *ta
 	if commandContext.OriginalError == nil {
 		logger.MarkSuccess("Cluster created!")
 	}
-	return nil
+	return &InstallPackageControllerTask{}
 }
 
 func (s *DeleteBootstrapClusterTask) Name() string {
 	return "delete-kind-cluster"
 }
+
+func (s *InstallPackageControllerTask) Run(ctx context.Context, commandContext *task.CommandContext) task.Task {
+	logger.Info("Installing curated packages controller on workload  cluster")
+	kubeConfig := kubeconfig.FromClusterName(commandContext.ClusterSpec.Cluster.Name)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere install command", "error", err)
+		return nil
+	}
+	chart := commandContext.ClusterSpec.VersionsBundle.VersionsBundle.PackageController.HelmChart
+	pc := curatedpackages.NewPackageControllerClient(deps.Helm, deps.Kubectl, kubeConfig, chart.Image(), chart.Name, chart.Tag())
+	err = pc.InstallController(ctx)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages controller on workload cluster; please install through eksctl anywhere install command", "error", err)
+		return nil
+	}
+	return nil
+}
+
+func (s *InstallPackageControllerTask) Name() string {
+	return "install-package-controller"
+}
+
+func installCuratedPackages(ctx context.Context, clusterName string, packagesLocation string) {
+	kubeConfig := kubeconfig.FromClusterName(clusterName)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
+	if err != nil {
+		logger.MarkFail("Error when installing curated packages on workload cluster; please install through eksctl anywhere create command", "error", err)
```

Comment:
This is called during create so I'm not sure the log message is particularly helpful. Ditto for the one below.

Reply:
There are two different commands in play, maybe I should clarify that with the message but the idea is that if this fails during cluster creation (the current command in execution), you have the ability to create it through create -f command.

Reply:
Sorry, I'm a little confused. The `create` cmd requires the `-f` flag so, if I'm not mistaken, they would've done that already and then received this log message that tells them to do the same thing. I feel like I'm missing something.

Reply:
As discussed offline, these are two commands.